blazor/ui-kit/getting-started.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Getting Started with Essential UI Kit for Blazor | Syncfusion
+description: Checkout and learn about getting started with the Essential UI Kit for Blazor in Syncfusion Essential Studio, along with more details.
+platform: Blazor
+control: UI Kit
+documentation: ug
+---
+
+# Getting Started with Essential UI Kit for Blazor
+
+Follow the steps below to get started with Essential UI Kit for Blazor. You can begin in one of two ways:
+
+- **Download the App from GitHub**: Get the full free source code and run the app locally to explore the blocks.
+- **View the Online Demo**: Alternatively, you can try out the blocks directly through the online demo to see them in action without needing to download anything.
+
+## Get Started by Downloading the App from GitHub
+
+### Step 1: Download and Open the App in Visual Studio Code
+
+Download the app from the GitHub [link](https://github.com/syncfusion/essential-ui-kit-for-blazor), and then open the same in Visual Studio Code to start working with it.
+
+![Launching the downloaded app](images/launching-the-downloaded-app.png)
+
+### Step 2: Open a New Terminal
+
+In Visual Studio Code, go to the **Terminal** menu and select **New Terminal** to open the integrated terminal.
+
+![Opening a new terminal](images/opening-a-new-terminal.png)
+
+### Step 3: Install Dependencies and Run the App
+Run the following commands one after the other in the terminal:
+
+1. **Restore dependencies**  
+   Run the command below to install all necessary packages related to the blocks, the Syncfusion Blazor components, and the sample browser.
+
+    ```bash
+    dotnet restore
+    ```
+
+    This will download and install all the dependencies listed in the **.csproj** file.
+
+2. **Serve the App Locally**  
+    Once the dependencies are installed, you can run the app locally by starting the Blazor development server. To do so, run the following command.
+    
+    ```bash
+    dotnet run
+    ```
+
+    This command will start the local development server, and you'll see an output in the terminal indicating the app is running. Typically, it will display a local host URL like this:
+
+    ![Local development server running](images/local-development-server-running.png)
+
+    To view the app in your browser, simply **Ctrl + Click** (or **Cmd + Click** on macOS) on the local host URL displayed in the terminal. This will open the app in your default browser, allowing you to interact with the blocks.
```

Comment:
Confirm the keyboard shortcut, both windows and iOS.

Reply:
Given keyboard shortcut is correct and confirmed